packages/cli/src/UserManagement/routes/index.ts

```diff
@@ -96,7 +97,7 @@ export function addRoutes(this: N8nApp, ignoredEndpoints: string[], restEndpoint
 			(req.method === 'DELETE' &&
 				new RegExp(`/${restEndpoint}/users/[^/]+`, 'gm').test(trimmedUrl)) ||
 			(req.method === 'POST' &&
-				new RegExp(`/${restEndpoint}/users/[^/]/reinvite+`, 'gm').test(trimmedUrl))
+				new RegExp(`/${restEndpoint}/users/[^/]+/reinvite`, 'gm').test(trimmedUrl))
```

Comment:
Note the regex change here. Assuming this was an oversight.

Reply:
can't we use this in the no-auth check?

Reply:
`/reinvite` is auth protected and only accessible by owner. Why do you mean to move it to the no-auth checks?

Reply:
I mean that we use a regex that excludes it but just details...